jenkins/osa_onmetal.functions.groovy

```diff
@@ -353,7 +353,7 @@ def start_api_uptime_tests() {
     sudo rm -f /usr/api.uptime.stop
     cd api_uptime/api_uptime
     python call_test.py -v
-    '''
+    ''' &
```

Comment:
This only handles the issue of running this in the background, but what about running it until we tell it to stop instead of running for a fixed amount of time?